build_run_docker.sh

```diff
@@ -60,6 +61,7 @@ NUM_API_NODES=0
 REPLICATION_FACTOR=1
 NODE_GROUPS=1
 RUN_BENCHMARK=
+VOLUMES_DIR=
```

Comment:
Rename to `LOCAL_VOLUMES_DIR`?

Reply:
Perhaps I should rename it `VOLUME_TYPE` instead and let it have two values `docker` and `local`.

Reply:
Did so, pushed.